src/main/scala/is/hail/table/Table.scala

```diff
@@ -102,13 +103,71 @@ object Table {
     Table(hc, rdd, typ, Array("id"))
   }
 
-  def apply(hc: HailContext, rdd: RDD[Row], signature: TStruct, key: IndexedSeq[String] = Array.empty[String],
-    globalSignature: TStruct = TStruct.empty(), globals: Annotation = Annotation.empty): Table = {
-    val rdd2 = rdd.mapPartitions(_.toRegionValueIterator(signature))
+  def apply(
+    hc: HailContext,
+    rdd: RDD[Row],
+    signature: TStruct
+  ): Table = apply(hc, rdd, signature, Array.empty[String])
+
+  def apply(
+    hc: HailContext,
+    rdd: RDD[Row],
+    signature: TStruct,
+    key: IndexedSeq[String]
+  ): Table = apply(hc, rdd, signature, key, TStruct.empty())
+
+  def apply(
```

Comment:
This seems kind of weird---being able to specify an arbitrary globalSignature but the globals are always an empty struct?